src/Maestro/DependencyUpdateErrorProcessor/.config/settings.Production.json

```diff
@@ -2,7 +2,5 @@
   "KeyVaultUri": "https://maestroprod.vault.azure.net/",
   "AppConfigurationUri": "https://maestroprod.azconfig.io/",
   "GithubUrl": "https://github.com/dotnet/core-eng",
-  "Owner": "dotnet",
```

Comment:
This doesn't seem related to the tests?

Reply:
I previously had separate keys for owner and repo, I was not parsing the github url.  Now I did so we no longer need this. So I cleaned this up.

Reply:
I recommend adding these kinds of changes to the description of your PR.

Looking at the title and the description, you only mention that you're adding test infrastructure, so it's hard to know if a change that is not related to that is intentional or a merge problem

Reply:
I will create  a new PR for this. Sorry for the confusion